src/components/button/ButtonCSS.js

```diff
@@ -130,4 +130,39 @@ export const ButtonCSS = `
     position: relative;
     overflow: hidden;
     cursor: pointer;
-`
\ No newline at end of file
+`
+
+export const ButtonNavigationCSS = `
+    position: absolute;
+    padding: 0;
+    background: none;
+    border: none;
+
+    font-family: 'HK Grotesk', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
```

Comment:
Noticed that all the font-sizes for the other buttons are `font-size: 1.5rem;` instead of the one taken from Figma (`font-size: 24px;`)

I think it should be 1.5rem instead? @ajo01 @phoenixthefirebird could someone confirm?

```suggestion
   font-size: 1.5rem;
```

Reply:
rem > px

Reply:
using rem should be fine